cv32e40s/sim/tools/xrun/covfile.tcl

```diff
@@ -51,4 +51,4 @@ set_covergroup -new_instance_reporting
 # Instances/modules to remove from coverage
 # For performance and to avoid spurious warnings, remove these modules from code coverage collection
 # ----------------------------------------------------------------------------------
-deselect_coverage -all -instance uvmt_cv32e40s_tb.iss_wrap...
\ No newline at end of file
+deselect_coverage -all -instance uvmt_cv32e40s_tb.imperas_dv...
```

Comment:
Could this interfere with the new soon-to-be coverage model?
If yes, then it will be very apparent when we can't find it, and then we can look back to this file?
If no, then there is no problem.

Reply:
This turns is to not gather coverage on the sv module that encompasses the ISS itself - its internal coverage mechanisms should not be affected. This behavior is the same as before, but the old module is removed and replaced by the new and thus this setting would have no effect and generate a warning in its old state.